tests/kafkatest/services/console_consumer.py

```diff
@@ -199,6 +200,9 @@ def pids(self, node):
     def alive(self, node):
         return len(self.pids(node)) > 0
 
+    def clean_shutdown(self, node):
```

Comment:
I don't think we need this method since tests can just access the `self.nodes_clean_shutdown` field directly.

Also, from line 234 I can see `nodes_clean_shutdown` is actually a list of numeric ids, so `nodes_clean_shutdown` is a somewhat misleading name.